### ISSUE TITLE
`prefer-v-bind-shorthand`

### ISSUE THREAD

**Issue by fisker** [vuejs/eslint-plugin-vue#2486]
**Please describe what the rule should do:**

Enforce shorthand `:id` over `:id="id"`

**What category should the rule belong to?**

[x] Enforces code style (layout)
[ ] Warns about a potential error (problem)
[ ] Suggests an alternate way of doing something (suggestion)
[ ] Other (please specify:)

**Provide 2-3 code examples that this rule should warn about:**

```vue
<!-- BAD -->
<img :id="id" :src="src" :alt="alt">

<!-- GOOD -->
<img :id :src :alt>
```

**Additional context**

https://blog.vuejs.org/posts/vue-3-4#v-bind-same-name-shorthand

**Comment by 7AXEL** [comment]
Check out this project: https://github.com/7AXEL/Web2APK

**Comment by waynzh** [comment]
Check out https://eslint.vuejs.org/rules/v-bind-style.html#samenameshorthand-always

**Comment by fisker** [comment]
Thank, sorry for missing.
